vault/logical_system.go

```diff
@@ -3363,6 +3363,12 @@ func (b *SystemBackend) pathInternalUIMountRead(ctx context.Context, req *logica
 
 	me := b.Core.router.MatchingMountEntry(ctx, path)
 	if me == nil {
+		// To be consistent with the case no client token was supplied, go through the motions of verifying authorization
```

Comment:
Do we also need to handle the equivalent case where the mount exists but replication filtering precludes using it?  (see checkReplicatedFiltering below)

Reply:
Yep, I think you're right, added.

Reply:
Why not call b.verifyAuthorizedMountAccess at the outset?  If we're going to return an error when it returns an error, why even bother looking up the mount entry or filtering?

Reply:
So that we return the error each code path meant to do if the verifyA..M..A.. succeeds.  In other words, only return a consistent error in the absence of a proper authenticated session, otherwise being more specific is okay.

Reply:
Hmm, yeah, that may be equivalent and less subject to timing attacks.  I'll revisit in the morning but that may be a good point.

Reply:
Your right I think, no harm in doing the token/entity check first.  Changed.